staging/src/k8s.io/component-base/metrics/testutil/metrics_test.go

```diff
@@ -54,58 +55,69 @@ func samples2Histogram(samples []float64, upperBounds []float64) Histogram {
 
 func TestHistogramQuantile(t *testing.T) {
 	tests := []struct {
+		name    string
 		samples []float64
 		bounds  []float64
 		q50     float64
 		q90     float64
 		q99     float64
 	}{
 		{
-			// repeating numbers
+			name:    "Repeating numbers",
 			samples: []float64{0.5, 0.5, 0.5, 0.5, 1.5, 1.5, 1.5, 1.5, 3, 3, 3, 3, 6, 6, 6, 6},
 			bounds:  []float64{1, 2, 4, 8},
 			q50:     2,
 			q90:     6.4,
 			q99:     7.84,
 		},
 		{
-			// random numbers
+			name:    "Random numbers",
 			samples: []float64{11, 67, 61, 21, 40, 36, 52, 63, 8, 3, 67, 35, 61, 1, 36, 58},
 			bounds:  []float64{10, 20, 40, 80},
 			q50:     40,
 			q90:     72,
 			q99:     79.2,
 		},
 		{
-			// the last bucket is empty
+			name:    "The last bucket is empty",
 			samples: []float64{6, 34, 30, 10, 20, 18, 26, 31, 4, 2, 33, 17, 30, 1, 18, 29},
 			bounds:  []float64{10, 20, 40, 80},
 			q50:     20,
 			q90:     36,
 			q99:     39.6,
 		},
+		{
+			name:    "The last bucket has +inf upper bound",
```

Comment:
```suggestion
			name:    "The last bucket has positive infinity upper bound",
```

Reply:
I'd prefer not to involve '+' here because this would make subtest names a little bit more complicated.  
When we run the subtest, we have to use slash like this:
```
-run=TestHistogramQuantile/The_last_bucket_has_\\+inf_upper_bound
```